proposals/lip-0042.md

```diff
@@ -86,18 +86,10 @@ The Reward module has the following internal function.
 
 This function is used to retrieve the reward of a block at a given height.
 
-##### Parameters
-
-`blockHeader`: A block header.
-
-##### Returns
-
-The reward for the block as a 64-bit unsigned integer.
-
 ##### Execution
 
 ```python
-getBlockReward(blockHeader):
+getBlockReward(blockHeader: BlockHeader) -> uint64:
```

Comment:
I think it would still be nice to define the `BlockHeader` type, similar to what we did here: https://github.com/LiskHQ/lips/pull/143/files#diff-855d735f598bef5639924d04e64a28a025082eec20f835526d33f68c5273625fR59